src/utils/requests.py

```
@@ -0,0 +1,94 @@
+"""Module to send requests to the model serving endpoint."""
+
+import random
+import time
+from concurrent.futures import ThreadPoolExecutor, as_completed
+
+import requests
+
+from src import logger
+
+
+from src import logger
```

Ruff (0.7.0)

Redefinition of unused `logger` from line 9

Remove definition: `logger`

(F811)